apps/meteor/client/views/room/contextualBar/VideoConference/VideoConfPopups/VideoConfPopup/IncomingPopup.tsx

```diff
@@ -35,19 +36,23 @@ const IncomingPopup = ({ id, room, position, onClose, onMute, onConfirm }: Incom
 	const { t } = useTranslation();
 	const { controllersConfig, handleToggleMic, handleToggleCam } = useVideoConfControllers();
 	const setPreferences = useVideoConfSetPreferences();
+	const roomName = useVideoConfRoomName(room);
 
 	const params = useMemo(() => ({ callId: id }), [id]);
 	const { phase, value } = useEndpointData('/v1/video-conference.info', { params });
 	const showMic = Boolean(value?.capabilities?.mic);
 	const showCam = Boolean(value?.capabilities?.cam);
 
+	const dialogLabel =
+		room.t === 'd' ? `${t('Incoming_call_from__roomName__', { roomName })}` : `${t('Incoming_call_from__roomName__', { roomName })}`;
```

Comment:
There seems to be something wrong here. The same string is being returned for both conditions.

Reply:
Indeed, thanks. Fixed now